coralproject/talk
# [next] GraphiQL Fixes

## What does this PR do?

Replaces standard express middleware with custom middleware addressing issue.

## How do I test this PR?

Set `ENABLE_GRAPHIQL=true`, and visit http://localhost:3000/tenant/graphiql, notice no errors in the console!
